src/examples/baby.svelte

```diff
@@ -48,9 +49,10 @@
     itemToString(item) {
       return item ? item.title : "";
     },
-    // onSelectedItemChange: ({selectedItem: newSelectedItem}) =>
-    //   setSelectedItem(newSelectedItem),
-    // });
+    selectItem: (item) => {
+      console.log(item);
+      // setSelectedItem(item),
```

Comment:
leaving this for future implementation. The internal `selectedItem` store keeps track of the value, this is for external use per this example in downshift: https://www.downshift-js.com/use-combobox#controlling-state